package.json

```diff
@@ -21,7 +21,7 @@
     "lint:types": "npm run types",
     "lint:versions": "node ./scripts/lint-versions.js",
     "prepare": "husky install",
-    "release": "changeset publish",
+    "release": "npm run build:types && changeset publish --tag lit2",
```

Comment:
Is it just me, but i dont see the task `build:types` anywhere.

Reply:
It isn't just you

Reply:
(fixed....)